tests/testthat/test-s3read.R

```diff
@@ -16,7 +16,7 @@ local({
       expect_error(s3read("key", cache = NA, serialize = FALSE))
     })  
     test_that("if cache is TRUE, it does not error", {
-      expect_equal(s3read("key", cache = TRUE, serialize = FALSE), "value")
+      expect_equal(s3read("key", cache = TRUE, serialize = FALSE), TRUE)
```

Comment:
The way the mocks are set up, https://github.com/robertzk/s3mpi/blob/master/R/s3read.r#L43 writes `TRUE` into `value`. Is this the desired behavior?